InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -165,15 +173,28 @@ public ShooterConfig() {
 
         public boolean useFancyOdometry = false;
 
-        public SimpleMatrix Ac_char = new SimpleMatrix(2, 2, true, new double[] {
-            0.8678678537600736, 0.0071917452743623786, 0.005903949588909425, 0.8723072041767262
+        // discrete time
+        public SimpleMatrix Ad_char = new SimpleMatrix(2, 2, true, new double[] {
+            0.8678678537600736, 0.0071917452743623786,
+            0.005903949588909425, 0.8723072041767262
         });
 
-        public SimpleMatrix Bc_char = new SimpleMatrix(2, 2, true, new double[] {
+        public SimpleMatrix Bd_char = new SimpleMatrix(2, 2, true, new double[] {
             0.050392385308988936, -0.004313538469407867,
             -0.002206154164037924, 0.047675029476658845
         });
 
+        // continuous time
+        public SimpleMatrix Ac_char = new SimpleMatrix(2, 2, true, new double[] {
+            -7.087195478354283, 0.413285738104402,
+            0.339280393075371, -6.832080740045777
+        });
+
+        public SimpleMatrix Bc_char = new SimpleMatrix(2, 2, true, new double[] {
+            2.702895517197959, -0.241632861263366,
+            -0.126961060623545, 2.551095849721741
+        });

```

Comment:
Is this drive config or navigation config? I'm confused. What are these values supposed to mean?